datalad_deprecated/ls_webui.py

```diff
@@ -27,6 +27,11 @@
 from datalad_deprecated.utils import safe_print
 from datalad.utils import with_pathsep
 from datalad.utils import Path
+import datalad.api as dl
+
+#remove the below 2 lines when NEEDS_CONTENT is already made false
+import datalad.metadata.extractors.datalad_core as dc
+dc.MetadataExtractor.NEEDS_CONTENT = False
```

Comment:
Got it, if the version is wrong should I just raise an error?

Reply:
no -- you just don't need to set it since we did that "upstream" for this extractor... so smth like

```suggestion
from datalad.support.external_versions import external_versions
if external_versions['datalad'] < '0.15.0':
    dc.MetadataExtractor.NEEDS_CONTENT = False
```